cmd/kube-controller-manager/app/options/options_test.go

```diff
@@ -71,6 +72,7 @@ func TestAddFlags(t *testing.T) {
 		"--cluster-signing-cert-file=/cluster-signing-cert",
 		"--cluster-signing-key-file=/cluster-signing-key",
 		"--concurrent-deployment-syncs=10",
+		"--concurrent-statefulset-syncs=5",
```

Comment:
this should be set to a non-default value to make sure the flag actually modifies the config field correctly